src/styles/tailwind/_base.scss

```diff
@@ -9448,6 +9456,20 @@
           user-select: text;
 }
 
+.select-all {
+  -webkit-user-select: all;
```

Comment:
Don't we have autoprefixer?

Reply:
Not sure, but this file is generated during the build process - I didn't add this directly.